src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -347,6 +351,212 @@ public async Task SendAsync_RequestContentNotDisposed()
             }
         }
 
+        [Fact]
+        public async Task GetStringAsync_Success()
+        {
+            string content = Guid.NewGuid().ToString();
+
+            await LoopbackServer.CreateClientAndServerAsync(
+                async uri =>
+                {
+                    using HttpClient httpClient = CreateHttpClient();
+
+                    string received = await httpClient.GetStringAsync(uri);
+                    Assert.Equal(content, received);
+                },
+                async server =>
+                {
+                    await server.AcceptConnectionSendResponseAndCloseAsync(content: content);
+                });
+        }
+
+        [Fact]
+        public async Task GetStringAsync_CanBeCanceled_AlreadyCanceledCts()
+        {
+            SemaphoreSlim onClientFinished = new SemaphoreSlim(0, 1);
```

Comment:
nit: In cases where you are doing a `new`, you can use `var` instead of the type name, i.e.

```c#
var onClientFinished = new SemaphoreSlim(0, 1);
```

Comment applies to other places with similar pattern.

The use of `var` is allowed when the type is obvious such as a `new` or an explicit cast.